pkg/didcomm/packer/anoncrypt/pack.go

```diff
@@ -40,13 +40,16 @@ type Packer struct {
 
 // New will create an Packer instance to 'AnonCrypt' payloads for a given list of recipients.
 // The returned Packer contains all the information required to pack and unpack payloads.
-func New(ctx packer.Provider, encAlg jose.EncAlg) *Packer {
+func New(ctx packer.Provider, encAlg jose.EncAlg) (*Packer, error) {
 	k := ctx.KMS()
+	if k == nil {
+		return nil, errors.New("anoncrypt: failed to create packer because KMS is empty")
```

Comment:
why would this happen?

The framework's defaults will inject the KMS. If the user injects a `nil` KMS then it renders the framework largely useless.... in which case I don't mind a panic. Although you can add a check to `WithKMS()` for this.

Reply:
Your assumption is that someone initialized the framework. If someone wants the use this `anoncrypt` without initializing the framework, then this is a gentle error to tell the user to setup a kms.